server/routes/auth/local.js

```diff
@@ -37,9 +37,15 @@ router.post('/login', (req, res, next) => {
         res.status(HttpStatus.INTERNAL_SERVER_ERROR).end();
       }
     } else if (user) {
-      req.login(user, () => {
+      req.login(user, async () => {
         if (req.accepts('html')) {
-          res.redirect('/auth/local/twoFactor');
+          const org = await user.getOrganization();
+          if (org.toJSON().isMfaEnabled) {
```

Comment:
You don't need to convert the object instance to JSON to access the attribute...

Reply:
I forgot to restart the docker containers when checking out the branches. I think I had that up when I was originally trying to debug why it was missing. Will fix